releases/release-1.30/release-notes/release-notes-draft.json

```diff
@@ -87,8 +180,8 @@
   },
   "118389": {
     "commit": "be77b0b82b01a3fc810118f095594ec8bdd3c3aa",
-    "text": "```\n\n#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:\n\n\u003c!--\nThis section can be blank if this pull request does not require a release note.\n\nWhen adding links which point to resources within git repositories, like\nKEPs or supporting documentation, please reference a specific commit and avoid\nlinking directly to the master branch. This ensures that links reference a\nspecific point in time, rather than a document that may change over time.\n\nSee here for guidance on getting permanent links to files: https://help.github.com/en/articles/getting-permanent-links-to-files\n\nPlease use the following format for linking documentation:\n- [KEP]: \u003clink\u003e\n- [Usage]: \u003clink\u003e\n- [Other doc]: \u003clink\u003e\n--\u003e",
```

Comment:
If this PR had the `/release-note-none` label, where is the text here coming from?

Reply:
I wonder if this happens when the release note block in the PR is left empty instead of adding "NONE" as requested in the instructions, or when they add it in the wrong place. For example: https://github.com/kubernetes/kubernetes/pull/121565

Reply:
This is probably a bug. We can remove this text (as you did) for now and submit an issue to [kubernetes/release](https://github.com/kubernetes/release) repo for debugging and fixing this (if it applies).

Reply:
https://github.com/kubernetes/release/issues/3516